pyne/mcnp.py

```diff
@@ -1998,202 +1998,223 @@ def _read_tallies(self, f):
-    exists within some meshtal file. Header information is stored as attributes
-    and the "mesh" attribute is a MOAB mesh with all result and relative error
-    data tagged. This class inherits from StatMesh, exposing all statistical
-    mesh manipulation methods.
-
-    Attributes
-    ----------
-    tally_number : int
-        The MCNP tally number. Must end in 4 (e.g. 4, 14, 214).
-    particle : string
-        Either "neutron" for a neutron mesh tally or "photon" for a photon mesh
-        tally.
-    dose_response : bool
-        True is the tally is modified by a dose response function.
-    x_bounds : list of floats
-        The locations of mesh vertices in the x direction.
-    y_bounds : list of floats
-        The locations of mesh vertices in the y direction.
-    z_bounds : list of floats
-        The locations of mesh vertices in the z direction.
-    e_bounds : list of floats
-        The minimum and maximum bounds for energy bins
-    mesh :
-        An PyMOAB core instance tagged with all results and
-        relative errors
-    tag_names : iterable
-        Four strs that specify the tag names for the results, relative errors,
-        total results, and relative errors of the total results.
-
-    Notes
-    -----
-    All Mesh/StatMesh attributes are also present via a super() call to
-    StatMesh.__init__().
-
-    """
-
-    def __init__(self, f, tally_number, tag_names=None, mesh_has_mats=False):
-        """Create MeshTally object from a filestream open to the second
-        line of a mesh tally header (the neutron/photon line). MeshTally objects
-        should be instantiated only through the Meshtal class.
-
-        Parameters
-        ----------
-        f : filestream
-            Open to the neutron/photon line.
+        Parameters:
+        -----------
+        f : str or filestream
+            Filestream of the meshtal file.
         tally_number : int
             The MCNP fmesh4 tally number (e.g. 4, 14, 24).
+        particle : str
+            The particle type, 'neutron' or 'photon'.
         tag_names : iterable, optional
             Four strs that specify the tag names for the results, relative
             errors, total results and relative errors of the total results.
             This should come from the Meshtal.tags attribute dict.
         mesh_has_mats : bool
              If false, Meshtally objects will be created without PyNE material
              objects.
-        """
-
-        if not HAVE_PYMOAB:
-            raise RuntimeError("PyMOAB is not available, "
-                               "unable to create Meshtally Mesh.")
-
-        self.tally_number = tally_number
-        self._read_meshtally_head(f)
-        self._read_column_order(f)
 
+        Returns:
+        --------
+        m : MeshTally object
+            The MeshTally object created from MCNP mesh tally of tally_number.
+        """
+        m = MeshTally()
```

Comment:
Please add a blank line after docstring for readability - perhaps also in other functions, even though some of the original functions didn't have this extra line.

Reply:
All the docstrings in `mesh.py` and `mcnp.py` are formatted according to PEP257:https://www.python.org/dev/peps/pep-0257/

Major changs:
- Add blank lines after mult-line docstrings
- Shorten some docstrings into one-line docstrings. (No blank line after one-line docstirngs).